pages/consulting/_comparison.html

```diff
@@ -0,0 +1,174 @@
+<div class="row row-page-header" style="margin-bottom: 60px;">
```

Comment:
A couple minor tweaks to the "How we're different" comparison table:

1. You can click on a row to expand it, but not to collapse it back down. Could you add the ability to collapse when you click on a row?
2. The up/down chevrons on the right side (^) seem backwards. They point up when the click actually expands the content downwards, and they point down when the click actually collapses things upwards. Could you reverse those?